apps/ewallet/lib/ewallet/gates/mint_gate.ex

```diff
@@ -39,8 +40,7 @@ defmodule EWallet.MintGate do
     "idempotency_token" => idempotency_token,
     "token_id" => token_id,
     "amount" => amount,
-    "description" => description,
-    "metadata" => metadata
+    "description" => description
```

Comment:
@unnawut The new logic:

- You can omit it from the request, nothing will be changed.
- If you set it, it cannot be set to `nil`, the API will send back an error.
- If you send a map, it will replace the current value.

Reply:
Sounds good! 👍